lib/node_modules/@stdlib/math/base/complex/divide/test/test.js

```diff
@@ -0,0 +1,283 @@
+'use strict';
+
+// MODULES //
+
+var tape = require( 'tape' );
+var isnan = require( '@stdlib/math/base/assert/is-nan' );
+var isinfinite = require( '@stdlib/math/base/assert/is-infinite' );
+var abs = require( '@stdlib/math/base/special/abs' );
+var pow = require( '@stdlib/math/base/special/pow' );
+var EPS = require( '@stdlib/math/constants/float64-eps' );
+var toBinaryString = require( '@stdlib/math/base/utils/float64-to-binary-string' );
+var cdiv = require( './../lib' );
+
+
+// FIXTURES //
+
+var quotients = require( './fixtures/julia/quotients.json' );
+
+
+// HELPERS
```

Comment:
This is a bad ad-hoc place for a helper. Is `test/utils/number-bits-correct.js` a better place?